.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Build & Publish
+
+on:
+  push:
+    tags: '**'
+
+  workflow_dispatch:
+
+jobs:
+
+  cd-job:
+    name: Continues Delivery
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: SCM Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Python & Poetry Environment
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.13.0
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
This workflow is not a matrix build

Reply:
Interesting. Seems I copied a file already containing an error in the source location, too. :slightly_smiling_face:

Reply:
Please see latest push